### ISSUE TITLE
How to train?

### ISSUE THREAD

**Issue by FelixZhang7** [Tushar-N/pytorch-resnet3d#2]

**Comment by Tushar-N** [comment]
Freeze batchnorm ([here](https://github.com/Tushar-N/pytorch-resnet3d/blob/master/models/resnet.py#L176)) if you're using small batch sizes, and train like you would a normal pytorch model. There's an untested train script [here](https://github.com/Tushar-N/pytorch-resnet3d/blob/master/utils/train.py) in case this isn't clear.
